__tests__/components/MapView.test.js

```diff
@@ -0,0 +1,21 @@
+import renderer from 'react-test-renderer';
+import MapView from 'js/components/MapView';
+import React from 'react';
+
+/**
+* Snapshot testing can be very useful if used correctly, this example just shows how to use
+* it but this is not the most useful snapshot, sometimes it is better to call it multiple
+* times inside a test and take snapshots of the component changing throughout some interactions,
+* Ex. snapshot default state, snapshot after click/mouseover, snapshot after mouseleave
+*/
+describe('MapView snapshot Tests', () => {
+
```

Comment:
Two minor changes here:

- Can we remove the `title` and `subtitle` properties here since the MapView does not accept these, might confuse someone who is newer to React.
- Can we change the text in test to say something like, 'Sample of snapshot testing for 3rd party components'